tasks.py

```diff
@@ -57,7 +57,7 @@ def _get_minimum_versions(dependencies, python_version):
     for dependency in dependencies:
         if '@' in dependency:
             name, url = dependency.split(' @ ')
-            min_versions[name] = f'{name} @ {url}'
+            min_versions[name] = f'{url}#egg={name}'
```

Comment:
@R-Palazzo Is there anything else that should be done here?

Reply:
I also updated the test here:
https://github.com/sdv-dev/CTGAN/blob/0d4708a3b0e64540785d776dc17285966eed477d/tests/test_tasks.py#L18

In this PR:
https://github.com/sdv-dev/CTGAN/pull/356